### PR TITLE
 Update based on last reviews

### PR DESCRIPTION
**Pull request by JunyiSun**

### CODE REVIEW

**Comment by JunyiSun** [comment]
Fixed #27

**Comment by evelinec** [comment]
@sishida would you help take a look at the new text changes? Thanks.

File: README.adoc [hunk 1]
```diff
@@ -115,7 +115,7 @@ The `<mpJwt>` element is required to configure the injection of the caller's JWT
 
 === Securing the system service
 
-Open the `start/backendServices/src/main/java/io/openliberty/guides/system/SystemResource.java` file and modify it to match the following code:
+Open the `start/backendServices/src/main/java/io/openliberty/guides/system/SystemResource.java` file and add roles-based access control. A simple way to proceed is to copy all the following code and replace the content of the file:
```
Comment (sishida) [inline README.adoc]: What about, "Simply copy all..."?

File: README.adoc [hunk 2]
```diff
@@ -133,7 +133,7 @@ Therefore, this service is properly secured.
 
 === Securing the inventory service
 
-In the `start/backendServices/src/main/java/io/openliberty/guides/inventory/` directory, open the `InventoryResource.java` file  and add roles-based access control as shown in the following code:
+Open the `start/backendServices/src/main/java/io/openliberty/guides/inventory/InventoryResource.java` file and add roles-based access control. A simple way to proceed is to copy all the following code and replace the content of the file:
```
Comment (sishida) [inline README.adoc]: What about, "Simply copy all..."?

File: README.adoc (hunk 1 above)
Comment (sishida) [reply README.adoc]: I would also use "contents" instead of "content."

**Pull request merged**